modules/vpc/main.tf

```diff
@@ -18,7 +18,6 @@
 	VPC configuration
  *****************************************/
 resource "google_compute_network" "network" {
-  provider                                  = google-beta
```

Comment:
I think it would be better to keep it as beta to reduce churn incase we want to add any beta related features in the future.